tasks/utils.ts

```diff
@@ -12,3 +16,141 @@ async function assertChainId({}, hre: HardhatRuntimeEnvironment) {
     );
   }
 }
+
+export async function deployDiamond(
+  {
+    ownerAddress,
+    diamondCutAddress,
+  }: {
+    ownerAddress: string;
+    diamondCutAddress: string;
+  },
+  {}: Libraries,
+  hre: HardhatRuntimeEnvironment
+) {
+  const factory = await hre.ethers.getContractFactory('Diamond');
+  const contract = await factory.deploy(ownerAddress, diamondCutAddress);
+  await contract.deployTransaction.wait();
+  console.log(`Diamond deployed to: ${contract.address}`);
+  return contract;
+}
+
+export async function deployFacet(
+  facetName: string,
+  libraries: Libraries,
+  hre: HardhatRuntimeEnvironment
+) {
+  const factory = await hre.ethers.getContractFactory(facetName, {
+    libraries,
+  });
+  const contract = await factory.deploy();
+  await contract.deployTransaction.wait();
+  console.log(`${facetName} deployed to: ${contract.address}`);
+  return contract;
+}
+
+export async function saveDeploy(
+  args: {
+    coreBlockNumber: number;
+    diamondAddress: string;
+    initAddress: string;
+    libraries: Libraries;
+  },
+  hre: HardhatRuntimeEnvironment
+) {
+  const isDev = hre.network.name === 'localhost' || hre.network.name === 'hardhat';
+
+  // Save the addresses of the deployed contracts to the `@darkforest_eth/contracts` package
+  const tsContents = `
+  /**
+   * This package contains deployed contract addresses, ABIs, and Typechain types
+   * for the Dark Forest game.
+   *
+   * ## Installation
+   *
+   * You can install this package using [\`npm\`](https://www.npmjs.com) or
+   * [\`yarn\`](https://classic.yarnpkg.com/lang/en/) by running:
+   *
+   * \`\`\`bash
+   * npm install --save @darkforest_eth/contracts
```

Comment:
```suggestion
   * npm install --save @dfdao/contracts
```
Will make README correct for installation of contracts